test/uk/gov/hmrc/personaldetailsvalidation/PersonalDetailsValidationResourceControllerSpec.scala

```diff
@@ -135,7 +135,7 @@ class PersonalDetailsValidationResourceControllerSpec
         "firstName is missing",
         "lastName is missing",
         "dateOfBirth is missing/invalid",
-        "nino is missing"
+        "at least nino or postcode needs to be supplied"
```

Comment:
I couldn't find test cases for postcode validation. Are they in some other file?

Reply:
I see you have written integration tests but I think unit tests are helpful when you need faster feedback from the tests.